src/main/resources/templates/fragments/betaBanner.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+    <head>
+            <title></title>
```

Comment:
empty title?

Reply:
It's standard practise with Thymeleaf fragments to keep valid html even if it's empty so that the fragment can be rendered on its own.